ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/converters/LdesMemberConverter.java

```diff
@@ -69,4 +78,10 @@ protected void writeInternal(Member member, HttpOutputMessage outputMessage)
 		body.write(outputStream.toString().getBytes());
 	}
 
+	@PostConstruct
```

Comment:
Forgot to delete ?

Reply:
Nope :) I'll explain in internal demo. It's an essential part actually